multihost_runner.py

```diff
@@ -200,23 +227,24 @@ def execute_main_command(main_command,slices, local_log_dir, run_name, zip_name,
     for worker_num in range(cur_slice.num_workers):
       output_filename = os.path.join(local_log_dir, f"output_slice_{cur_slice.slice_num:04d}_worker_{worker_num:04d}.txt")
       output_logs.append(output_filename)
-      mkdir_command = f"mkdir -p {run_name}"
-      mv_zip_command = f"mv {zip_name} {run_name}"
-      cd_command = f"cd {run_name}"
+      mkdir_command = f"mkdir -p {args.RUN_NAME}"
```

Comment:
Is there a chance we will call this function from another module? Calling this as `args.RUN_NAME` won't make it reusable by other modules.

Reply:
discussed offline